src/protocols/LoRaWAN/LoRaWANBands.cpp

```diff
@@ -20,16 +20,18 @@ const LoRaWANBand_t* LoRaWANBands[RADIOLIB_LORAWAN_NUM_SUPPORTED_BANDS] = {
 const LoRaWANBand_t EU868 = {
   .bandNum = BandEU868,
   .bandType = RADIOLIB_LORAWAN_BAND_DYNAMIC,
-  .payloadLenMax = {  59,  59,  59, 123, 230, 230, 230, 230,   0,   0,   0,   0,   0,   0,   0 },
+  .payloadLenMax = {  59,  59,  59, 123, 250, 250, 250, 250,  58, 123,  58, 123,   0,   0,   0 },
+  .payloadLenRel = {  59,  59,  59, 123, 230, 230, 230, 230,  58, 123,  58, 123,   0,   0,   0 },
   .powerMax = 16,
   .powerNumSteps = 7,
   .dutyCycle = 36000,
   .dwellTimeUp = 0,
   .dwellTimeDn = 0,
+  .txParamSupported = false,
   .txFreqs = {
-    { .enabled = true, .idx = 0, .freq = 868.100, .drMin = 0, .drMax = 5},
-    { .enabled = true, .idx = 1, .freq = 868.300, .drMin = 0, .drMax = 5},
-    { .enabled = true, .idx = 2, .freq = 868.500, .drMin = 0, .drMax = 5},
+    { .enabled = true, .idx = 0, .freq = 8681000, .drMin = 0, .drMax = 5, .dr = 5 },
+    { .enabled = true, .idx = 1, .freq = 8683000, .drMin = 0, .drMax = 5, .dr = 5 },
+    { .enabled = true, .idx = 2, .freq = 8685000, .drMin = 0, .drMax = 5, .dr = 5 },
```

Comment:
I know this has been here for a while, but I only noticed it now - why is the member `idx` needed? It seems to be just incrementaly increasing for each entry, so it can be inferred if needed.

Reply:
Welcome to the Dark Side, where channel index is not equal to channel slot. For example the US915 band: the default subband 2 uses channels 8-15 and 65. We're not allocating 72 (80) channel slots (16 is the mandatory / recommended minimum), so we have to keep track of index too.